Source/DafnyStandardLibraries/src/dfyconfig-arithmetic.toml

```diff
@@ -0,0 +1,29 @@
+
+includes = ["**/Arithmetic/*.dfy", "**/Arithmetic/**/*Internals.dfy"]
+# *Nonlinear.dfy in Arithmetic/Internals/ need non-linear arithmetic and are subject to dfyconfig.toml
+
+[options]
+disable-nonlinear-arithmetic = true
+
+# These are necessary since Internals/*Nonlinear.dfy is subject to a separate toml file
+
+library = [
+  "DafnyStdLibs/Arithmetic/Internals/DivInternalsNonlinear.dfy",
+  "DafnyStdLibs/Arithmetic/Internals/ModInternalsNonlinear.dfy",
+  "DafnyStdLibs/Arithmetic/Internals/MulInternalsNonlinear.dfy"
+]
+
+# (Ideally we would find a way to share configuration between these two project files)
+
+# Options that help support more values of consuming context options
+# (mostly turning on any more restrictive verification)
+track-print-effects = true
+enforce-determinism = true
+reads-clauses-on-methods = true
+
+# Options important for sustainable development
+# of the libraries themselves.
+resource-limit = 1000
+#warn-as-errors = true
```

Comment:
I would keep this one on even if it's reasonable to remove the assumptions ones, since otherwise end users may see other warnings when they flip on `--standard-libraries`

Reply:
Should be back in the latest version!